### PR TITLE
fix(components/lookup): reset focus for search result controls when new text entered in input

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
[AB#2673842](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2673842)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1727/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1727/storybooks/ag-grid-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1727/storybooks/data-manager-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1727/storybooks/lookup-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1727/storybooks/phone-field-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1727/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1727/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1727/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1727/playground/)

File: libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.spec.ts [hunk 1]
```diff
@@ -1698,6 +1698,43 @@ describe('Autocomplete component', () => {
         expect(getSearchResultsContainer()).toBeNull();
       }));
 
+      it('should reset the value and not emit the add event if enter is pressed when the add button is focused but space is then entered', fakeAsync(() => {
```
Comment (Blackbaud-SteveBrush) [inline libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.spec.ts]: This message communicates too much of the inner workings of the autocomplete component; if those internal workings changed, we'd have to remember to change this test's description. Could you describe the scenario being tested more generically? 

I would imagine this `it` description would have similar wording to the commit message, as well.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
LGTM. For the commit message, I would explain the expected behavior instead of saying what the code is no longer doing. Something like, `fix(components/lookup): reset focus for search result controls when new text entered in input`.

File: libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.spec.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.spec.ts]: Done

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**